.github/actions/templates/avm-validateModuleDeployment/action.yml

```diff
@@ -269,6 +274,11 @@ runs:
               resourceLocation = '${{ steps.get-resource-location.outputs.resourceLocation }}'
             }
           }
+          if($moduleTemplatePossibleParameters -contains 'baseTime') {
+            $functionInput.AdditionalParameters += @{
+              baseTime = (Get-Date).ToString('u') # Setting base time explcitly so that any potential retry will reuse the same timestamp
```

Comment:
```suggestion
              baseTime = (Get-Date).ToString('u') # Setting base time explicitly so that any potential retry will reuse the same timestamp
```